### PR TITLE
test(extension): [LW-8666] add e2e tests for staking empty states & start staking

### PR DESCRIPTION
**Pull request by ljagiela**
# Allure report
`allure-report-publisher` generated test report!

**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/2756/6690676173/index.html) for [6871a146](https://github.com/input-output-hk/lace/pull/681/commits/6871a146b5e5d82170203f49af7a26c33b3fb37c)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 30     | 2      | 0       | 0     | 32    | ❌     |

### CODE REVIEW

File: packages/e2e-tests/src/steps/multidelegationSteps.ts [hunk 1]
```diff
@@ -254,3 +259,27 @@ When(/^I'm on a delegation flow "([^"]*)"$/, async (delegationStep: string) => {
...
+
+Then(/^I see Start Staking page in (extended|popup) mode$/, async (mode: 'extended' | 'popup') => {
+  const cardanoBalance = String(await TokensPageObject.loadTokenBalance('Cardano'));
+  await StartStakingPageAssert.assertSeeStartStakingPage(cardanoBalance, mode);
+});
+
+Given(/^I am on Start Staking page in (extended|popup) mode$/, async (mode: 'extended' | 'popup') => {
+  await TokensPageObject.waitUntilCardanoTokenLoaded();
+  await TokensPageObject.saveTokenBalance('Cardano');
+  await localStorageInitializer.disableShowingMultidelegationBetaBanner();
```
Comment (oldGreg5) [inline packages/e2e-tests/src/steps/multidelegationSteps.ts:271]: Don't we want to use step for this for consistency? 
'And I disable showing Multidelegation beta banner'
We wanted to refactor this step later and make it part of wallet setup or some other appropriate place to disable it for all, unless we explicitly want it to be shown.

**Review by oldGreg5 — CHANGES_REQUESTED** [review]

**Review by oldGreg5 — APPROVED** [review]

File: packages/e2e-tests/src/steps/multidelegationSteps.ts [hunk 2]
```diff
@@ -254,3 +259,27 @@ When(/^I'm on a delegation flow "([^"]*)"$/, async (delegationStep: string) => {
...
+  await mainMenuPageObject.navigateToSection('Staking', mode);
+  const cardanoBalance = String(await TokensPageObject.loadTokenBalance('Cardano'));
+  await StartStakingPageAssert.assertSeeStartStakingPage(cardanoBalance, mode);
+});
+
+Then(/^I click Get Started step '(1|2)' link$/, async (linkNumber: '1' | '2') => {
```
Comment (wklos-iohk) [inline packages/e2e-tests/src/steps/multidelegationSteps.ts:277]: I'd suggest to wrap `Get Started`  with double quotes to distinguish that label and remove single quotes that are wrapping step number as here are only 2 predefined options.

File: packages/e2e-tests/src/steps/multidelegationSteps.ts (hunk 1 above)
Comment (ljagiela) [reply packages/e2e-tests/src/steps/multidelegationSteps.ts:271]: There is no way now to workaround this. I think we will handle it in separate PR.
